### ISSUE TITLE
[Bug][a11y] Talkback is blocked when the search/address bar is focused

### ISSUE THREAD

**Issue by SoftVision-LorandJanos** [mozilla-mobile/fenix#27299]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Launch the Firefox application and go on homepage.
3. Focus on the search/address bar.
4. Tap on any other item on homescreen.

### Expected behaviour

The selected item is focused.

### Actual behaviour

Talkback remains blocked on the search/address bar.

### Device name

Google Pixel 4

### Android version

Android 12

### Firefox release type

Firefox Nightly

### Firefox version

107.0a1

### Additional information

https://user-images.githubusercontent.com/48829350/194266432-96ed301e-f7e1-4415-8366-8848eec7e562.mp4

- This is reproducible even after the keyboard is hidden.
- Workaround would be to go back from the device back button.
- While "Show tap" was activated on the video through Developer options, it wasn't showing for some reason, but I tried to tap on the items from the homescreen.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21737)

**Comment by DiegoARL38** [comment]
Possibly a duplicate of #20847.

See also #20491.

**Comment by Mugurell** [comment]
For me navigating to swipes works for the search UX - toolbar + the "Scan" and "Search engine" button.
But indeed tapping homescreen items does not select them.
@SoftVision-LorandJanos  Can you confirm?

https://user-images.githubusercontent.com/11428869/194511025-7e56aaed-9828-43ed-bc9a-ab8a9b95399c.mp4

**Comment by Mugurell** [comment]
Thank you @SoftVision-LorandJanos and @DiegoARL38 !
Than indeed this seems like a duplicate of https://github.com/mozilla-mobile/fenix/issues/20847 which I see was already prioritized.
